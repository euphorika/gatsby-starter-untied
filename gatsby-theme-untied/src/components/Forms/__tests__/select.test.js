import React from 'react'
import renderer from 'react-test-renderer'
import Select from '../select'

describe('Select', () => {
  it('renders correctly', () => {
    const select = (
      <Select name="Select">
        <option value="" />
      </Select>
    )

    const tree = renderer.create(select).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
