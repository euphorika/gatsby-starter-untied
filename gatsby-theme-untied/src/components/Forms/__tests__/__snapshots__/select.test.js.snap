// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select renders correctly 1`] = `
<div
  className="element-container css-0"
>
  <div
    className="select-box"
  >
    <select
      className="element"
      name="Select"
    >
      <option
        value=""
      />
    </select>
    <span
      className="after-form-element"
    />
  </div>
</div>
`;

exports[`Select renders required correctly 1`] = `
<div
  className="element-container css-0"
>
  <div
    className="select-box required"
  >
    <select
      className="element"
      name="Select"
      required={true}
    >
      <option
        value=""
      />
    </select>
    <span
      className="after-form-element"
    />
  </div>
</div>
`;
